corehq/ex-submodules/couchexport/writers.py

```diff
@@ -360,14 +361,8 @@ class Excel2007ExportWriter(ExportWriter):
     max_table_name_size = 31
 
     def _init(self):
-        try:
-            import openpyxl
-        except ImportError:
-            raise Exception("It doesn't look like this machine is configured for "
-                            "excel export. To export to excel you have to run the "
-                            "command:  easy_install openpyxl")
-
-        self.book = openpyxl.Workbook(optimized_write=True)
+        self.book = openpyxl.Workbook()
+        self.book.remove(self.book.worksheets[0])
```

Comment:
I'm not sure what "default sheet" from the commit means. Maybe this could use a comment to say why it's necessary?

Reply:
Updated - when I passed the ```write_only``` kwarg a default sheet isn't created anymore